src/sentry/scim/endpoints/users.py

```diff
@@ -0,0 +1,156 @@
+from django.conf import settings
+from django.db import transaction
+from django.db.models import Q
+from rest_framework.response import Response
+
+from sentry import roles
+from sentry.api.bases.organizationmember import OrganizationMemberEndpoint
+from sentry.api.endpoints.organization_member_details import OrganizationMemberDetailsEndpoint
+from sentry.api.endpoints.organization_member_index import OrganizationMemberSerializer
+from sentry.api.paginator import GenericOffsetPaginator
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.organization_member import OrganizationMemberSCIMSerializer
+from sentry.models import AuditLogEntryEvent, AuthIdentity, InviteStatus, OrganizationMember
+from sentry.signals import member_invited
+
+from .constants import SCIM_409_USER_EXISTS
+from .utils import SCIMEndpoint, parse_filter_conditions
+
+ERR_ONLY_OWNER = "You cannot remove the only remaining owner of the organization."
+
+
+class OrganizationSCIMUserDetails(SCIMEndpoint, OrganizationMemberEndpoint):
+    def get(self, request, organization, member):
+        context = serialize(member, serializer=OrganizationMemberSCIMSerializer())
+        return Response(context)
+
+    def patch(self, request, organization, member):
+        for operation in request.data.get("Operations", []):
+            # we only support setting active to False which deletes the orgmember
+            if operation["value"]["active"] is False:
+                audit_data = member.get_audit_log_data()
+                if OrganizationMemberDetailsEndpoint._is_only_owner(member):
```

Comment:
Couldn't this be `self._is_only_owner(member)` as you're inheriting from the OrganizationMemberEndpoint?

Reply:
the `OrganizationMemberDetailsEndpoint` doesn't actually use `OrganizationMemberEndpoint ` -- there is definitely a future refactor to be done so it does -- in the meantime i just decided to grab the method directly. I could also inherit from that endpoint additionally to use the method but felt weird to me.

I suppose we could also make is_only_owner a property on the organizationmember model.